src/index.html

```diff
@@ -11,7 +11,170 @@
+            class="field"
+            required
+          >
+        </label>
+        <label
+          class="form-field"
+        >
+          Full date of birth:
+          <input
+            type="date"
+            name="date">
+        </label>
+        <label
+          class="form-field"
+        >
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="tearm"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <label
+        class="form-field"
+        >
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            class="field"
+          >
+        </label>
+        <label
+        class="form-field"
+        >
+          Password:
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="10"
+            class="field"
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="catLover"
+              value="yes"
+            > Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="catLover"
+              value="no"
+            > No
+          </label>
+        </div>
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            value="#080808"
+          >
+        </label>
+        <label class="form-field">
+          What time do you go to bad?
+          <input
+            type="time"
+            name="time"
+            class="field"
+            min="00:00:00"
+          >
+        </label>
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select name="brand" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="form-field">
+          How do you rate our work?
+          <input type="range"
+            name="rate"
+            min="0"
+            max="10"
+            value="0"
+            class="field"
+          >
+        </label>
+      </fieldset>
+      <fieldset>
+        <legend>Additional info:</legend>
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="comments"
+            class="field"
+          ></textarea>
+        </label>
+        <label class="form-field">
+          Would you recommend us?
+          <select name="recomend">
+            <option
+              value="yes" selected>yes</option>
```

Comment:
[checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 3rd item
```suggestion
            <option value="yes" selected>
              yes
            </option>
```